app/controllers/paths_controller.rb

```diff
@@ -6,7 +6,9 @@ def show
   end
 
   def index
-    @paths = Path.all
+    # The filter on here is temporary until the path is removed and the students
+    #  enrolled in it are migrated to another path.
+    @paths = Path.all.filter { |path| path.title != 'Front End Only' }
```

Comment:
You could use reject here instead, it does the inverse of filter and often reads better when you need to use not equals in the block:

`Path.all.reject { |path| path.title == 'Front End Only' }`


I don't expect you to change it to that though, there is nothing wrong with this implementation. Just thought it would be neat thing to show you.